### DISCUSSION TITLE
Change the content/docs folder name

### DISCUSSION THREAD

**Discussion by DataHawker** [fuma-nama/fumadocs#850] [General]
Hi. I was able to fix a lot of issues with multi-docs, Nav (cleaner route groups fixed it), and so much more. 

My one issue is I can change the docs/[slug] to pubs/[slug] and update the source.ts with the new base url but when I change the content/docs to content/pubs - I get 404 errors. I do not see a clear file that directs from docs to a new name. 

Also, I want to put a user avatar next to the theme switch on the top nav and profile inside the three dots on mobile. 

Thanks!

**Comment by fuma-nama** [comment]
For customising how we handle your content, you can see the docs of your content source. Fumadocs MDX has a "Configuration" section for this, you can pass a dir property to your collections.

If you are using Content Collections, make sure to read their docs

**Comment by Gitstar-OC** [comment]
Is this issue solved ?
